Release_Guideline.md

```diff
@@ -0,0 +1,40 @@
+# Release Guidelines
+
+This document captures guidelines to follow when considering a new release of the BIDS specification.
+
+## Background
+BIDS has generally followed a semantic versioning (semver) approach.
+The canonical semver text can be found at [https://semver.org/](https://semver.org/), and an adaptation to documents can be found at [https://semverdoc.org/](https://semver.org/).
```

Comment:
The second link was targeting the wrong place, but Markdown should render bare URLs just fine.

```suggestion
The canonical semver text can be found at https://semver.org/, and an adaptation to documents can be found at https://semverdoc.org/.
```